generators/liquibase/generator.ts

```diff
@@ -217,6 +217,7 @@ export default class LiquibaseGenerator extends BaseEntityChangesGenerator {
               this.prepareChangelog({
```

Comment:
We should avoid mutating databaseChangelog up in the other `this.prepareChangelog` call too.

Reply:
ah, true :-)